apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/_components/DocumentLogs/DocumentLogsTable.tsx

```diff
@@ -32,12 +32,8 @@ function countLabel(count: number) {
   return `${count} logs`
 }
 
-type DocumentLogRow = DocumentLogWithMetadataAndError & {
-  realtimeAdded?: boolean
-}
-
 type Props = {
-  documentLogs: DocumentLogRow[]
+  documentLogs: DocumentLogWithMetadataAndError[]
```

Comment:
This is fantastic. Now, what comes from the API is known. And we can't lie to typescript inferred types. So our little hack of mutating collection objects to put the real-time flash keeps inside this component. However, the external world knows the standard collection of logs with errors is the prop received.